nion/swift/model/Graphics.py

```diff
@@ -1074,6 +1085,19 @@ def nudge(self, mapping: CoordinateMappingLike, delta: Geometry.FloatSize) -> No
     def draw(self, ctx: DrawingContextLike, ui_settings: UISettings.UISettings, mapping: CoordinateMappingLike, is_selected: bool = False) -> None:
         raise NotImplementedError()
 
+def make_rectangle_mask(data_shape: DataAndMetadata.ShapeType, center: NormPointType, size: NormSizeType, rotation: float) -> DataAndMetadata._ImageDataType:
```

Comment:
Can you move this to the top of the file? Just before `get_closest_point_on_line` would be perfect.